tests/observability-test-utils/src/main/ballerina/Ballerina.toml

```diff
@@ -4,7 +4,7 @@ name = "testobserve"
 version = "0.0.0"
 
 [[platform.java11.dependency]]
-path = "../libs/observability-test-utils-2.0.0-Preview7.jar"
+path = "../libs/observability-test-utils-2.0.0-Preview8-SNAPSHOT.jar"
```

Comment:
do we need to commit these toml changes

Reply:
Most of the lang modules and other modules seem to have been committed with the new version. I can revert if this shouldn't be done.